addon/templates/components/frost-chart-svg-plot-stacked-bar.hbs

```diff
@@ -0,0 +1,3 @@
+{{#each _bars key='@index' as |bar|}}
```

Comment:
What purpose does `key='@index'` serve?

Reply:
Good question... that was a copy/paste from existing code, but I don't think it's actually doing anything. I'll remove it.